rotkehlchen/tests/exchanges/test_kraken.py

```diff
@@ -69,6 +69,8 @@ def test_coverage_of_kraken_balances(kraken):
     got_assets.remove('FLOWH.S')
     got_assets.remove('FLOWH')  # what is FLOWH?
     got_assets.remove('ADA.S')
+    got_assets.remove('SOL.S')
+    got_assets.remove('KSM.P')  # Couldn't find what this is
```

Comment:
I think I know! It's good to ignore like this for now. It's probably kusama bonded for parachains: 

https://www.kraken.com/en-gb/learn/parachain-auctions

Can you change the comment?